### ISSUE TITLE
Package conflict with pytables

### ISSUE THREAD

**Issue by kcrkor** [conda-forge/geopandas-feedstock#12]
I cannot install geopandas after installing blaze (which requires a newer version of pytables). 

```
The following specifications were found to be in conflict:                                                                                     
  - geopandas                                                                                                                                  
  - pytables -> hdf5 1.8.13|1.8.14|1.8.15.1|1.8.16|1.8.9                                                                                       
  - pytables -> numexpr 2.0*|2.1*|2.2*|2.3*                                                                                                    
  - pytables -> numpy 1.6*|1.7*|1.8*|1.9*                                                                                                      
  - pytables -> python 2.6*|2.7*|3.3*|3.4*  
```

N.B.: If geopandas is installed first then blaze is installed, geopandas conflicts with itself similar to #1

**Comment by ocefpaf** [comment]
@kcrkor can you tell me the Python version and platform that you tried this?

**Comment by kcrkor** [comment]
@ocefpaf this is happening with in my docker image which is based on [jupyter/all-spark-notebook](https://github.com/jupyter/docker-stacks/tree/master/all-spark-notebook). If I want to install blaze I cannot do so until I remove geopandas from the image. Reinstalling geopandas afterwards doesn't work either.

**Comment by ocefpaf** [comment]
What is the python version? Can you send me the results of `conda info` and `conda list`?

**Comment by jakirkham** [comment]
So, I tried to reproduce the same in a vanilla `jupyter/all-spark-notebook` and this is what I have found.

```
$ conda install -c conda-forge geopandas
Fetching package metadata ...........
Solving package specifications: ....

The following specifications were found to be in conflict:
  - geopandas
  - jpeg 8*
Use "conda info <package>" to see the dependencies for each package.
```

As R seemed to want `jpeg` version `8` and we were unable to track down why, it seemed easiest to just pin it so that we could fix the stack. Now I'm not totally clear on where `jpeg` comes into the picture with `geopandas`. However, conda-forge uses `jpeg` version `9` everywhere. So I'm not surprised it would cause an issue.
